contracts/deploy/011_ousd_fix.js

```diff
@@ -62,19 +60,21 @@ const fixOUSD = async () => {
     log("Upgraded OUSD to new implementation");
   }
 
-  console.log(`${deployName} deploy done.`);
```

Comment:
nit: Why removing this line? Not a big deal but I found it useful to know when a deployment is finished running.

Reply:
This line was duplicated. Inside this function and also inside the main function.

Reply:
Got it. Thanks.